packages/contract-artifacts/artifacts/OrderValidator.json

```diff
@@ -231,6 +231,7 @@
                 "type": "constructor"
             }
         ],
+        "devdoc": { "methods": { } },
```

Comment:
Gasp! :)

Reply:
lol! I think WETH9 also doesn't have any devdocs. `OrderValidator` needs to be removed anyway. I'll do that in a separate PR.